pkgs/applications/science/electronics/digital/default.nix

```diff
@@ -3,17 +3,18 @@
 }:
 
 let
+  pname = "digital";
   pkgDescription = "A digital logic designer and circuit simulator.";
   version = "0.30";
   buildDate = "2023-02-03T08:00:56+01:00"; # v0.30 commit date
 
   desktopItem = makeDesktopItem {
     type = "Application";
-    name = "Digital";
-    desktopName = pkgDescription;
+    name = pname;
+    desktopName = "Digital";
     comment = "Easy-to-use digital logic designer and circuit simulator";
-    exec = "digital";
-    icon = "digital";
+    exec = pname;
+    icon = pname;
```

Comment:
It is not preferred to reference them as `pname`, see #277994.

Reply:
Oh, this package also uses `pname` as the binary and icon name, so it's kind of consistent here...

Reply:
It's, well, okay, 🫠